docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
Adding here so that markdown link checker does not complain.

Reply:
I should have clarified. This file stems from the `main` branch but added here because I reference the `adr-013-equivocation-slashing.md` file from the `docs/docs/features/slashing.md` file. Without having the file in this PR, the markdown link checker would have complained.